### ISSUE TITLE
VMap Ingress Module for FMS

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#35]
## User story
1. As a SD
2. I want to develop a VMap Ingress module
3. So that I can read scanned LIDAR map file and convert it to FMS Python graph

## Acceptance criteria
* Be able to read scanned LIDAR map file
* Be able to convert it to FMS Python graph

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by UmangBR** [comment]
Required because we need to read the recorded map file (Lidar map) into a format we can store in the FMS (graph) to later calculate routes
